plugins/win-capture/graphics-hook/obs-vulkan32.json

```diff
@@ -0,0 +1,82 @@
+{
+    "file_format_version": "1.1.2",
+    "layer": {
+        "name": "VK_LAYER_OBS_HOOK",
+        "type": "GLOBAL",
+        "library_path": ".\\graphics-hook32.dll",
+        "api_version": "1.1.114",
+        "implementation_version": "1",
+        "description": "Open Broadcaster Software hook",
+        "functions": {
+            "vkNegotiateLoaderLayerInterfaceVersion": "OBS_Negotiate",
+            "vkGetInstanceProcAddr": "OBS_GetInstanceProcAddr",
+            "vkGetDeviceProcAddr": "OBS_GetDeviceProcAddr"
+        },
+        "pre_instance_functions": {
+            "vkEnumerateInstanceExtensionProperties": "OBS_EnumerateInstanceExtensionProperties"
+        },
+        "instance_extensions": [
+            {
+                "name": "VK_KHR_external_memory_capabilities",
+                "spec_version": "1.1",
+                "entrypoints": [
+                    "OBS_ExtMemCapBegin",
+                    "OBS_ExtMemCapEnd"
+                ]
+            },
+            {
+                "name": "VK_KHR_get_physical_device_properties2",
+                "spec_version": "1",
+                "entrypoints": [
+                    "OBS_GetPhyDevProp2Begin",
+                    "OBS_GetPhyDevProp2End"
+                ]
+            }
+        ],
+        "device_extensions": [
+            {
+                "name": "VK_KHR_external_memory_win32",
+                "spec_version": "1",
+                "entrypoints": [
+                    "OBS_ExtMemWin32Begin",
+                    "OBS_ExtMemWin32End"
+                ]
+            },
+            {
+                "name": "VK_KHR_external_memory",
+                "spec_version": "1",
+                "entrypoints": [
+                    "OBS_ExtMemBegin",
+                    "OBS_ExtMemEnd"
+                ]
+            },
+            {
+                "name": "VK_KHR_dedicated_allocation",
+                "spec_version": "1",
+                "entrypoints": [
+                    "OBS_DedAllocBegin",
+                    "OBS_DedAllocEnd"
+                ]
+            },
+            {
+                "name": "VK_KHR_get_memory_requirements2",
+                "spec_version": "1",
+                "entrypoints": [
+                    "OBS_GetMemReq2Begin",
+                    "OBS_GetMemReq2End"
+                ]
+            },
+            {
+                "name": "VK_KHR_bind_memory2",
+                "spec_version": "1",
+                "entrypoints": [
+                    "OBS_BindMem2Begin",
+                    "OBS_BindMem2End"
+                ]
+            }
+        ],
+        "disable_environment": {
+            "DISABLE_VULKAN_OBS_CAPTURE": "1"
+        }
+    }
+}
```

Comment:
Seems to be very different from obs-vulkan64.json. Should probably align the files.